src/EFCore.PG/Storage/Internal/NpgsqlExecutionStrategy.cs

```diff
@@ -9,12 +9,10 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal
 {
     public class NpgsqlExecutionStrategy : IExecutionStrategy
     {
-        private ExecutionStrategyDependencies Dependencies { get; }
+        ExecutionStrategyDependencies Dependencies { get; }
```

Comment:
A good candidate to be a field.

Reply:
Just following EF Core conventions as usual...